app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -316,6 +317,17 @@ public String prependEdit(String editToken, String processedPageContent, String
                 .getString("/api/flow-parsoid-utils/@content"));
     }
 
+    @Override
+    @NonNull
+    public String getWikiText(PageTitle title) throws IOException {
```

Comment:
javadoc explaining (in just a few words) what wikitext is, to avoid misunderstandings.